lib/smoke_detector/version.rb

```diff
@@ -1,3 +1,3 @@
 module SmokeDetector
-  VERSION = "1.1.5"
+  VERSION = "1.1.6"
```

Comment:
1.1.5 was tagged on the master (rails 4) branch at one point... this avoids collisions.